src/Ookii.Dialogs.Wpf/Ookii.Dialogs.Wpf.nuspec

```diff
@@ -26,8 +26,9 @@
     </frameworkAssemblies>
   </metadata>
   <files>
-    <file src="bin\$configuration$\Ookii.Dialogs.Wpf.dll" target="lib\net45\" />
-    <file src="bin\$configuration$\Ookii.Dialogs.Wpf.pdb" target="lib\net45\" />
-    <file src="bin\$configuration$\Ookii.Dialogs.Wpf.xml" target="lib\net45\" />
+    <file src="bin\$configuration$\net45\Ookii.Dialogs.Wpf.dll" target="lib\net45\" />
```

Comment:
This looks good assuming we'll continue to create the NuGet package using the NuGet CLI. I'm just wondering if we shouldn't move to `dotnet pack` on the `.csproj` now that we are going to the new SDK project format. What do you think?